docs/implementing.rst

```diff
@@ -120,3 +120,36 @@ Users who only have a smartphone will have difficulty scanning the QR code
 during setup. You can directly show the secret key within the QR code in text
 form during setup by providing your own ``two_factor/core/setup.html`` template
 and using the ``secret_key`` context variable.
+
+Custom Templates
+--------------------------------
+To create a custom template, add a template called ``two_factor/_base.html``.
+As a bare minimum, its contents should contain a ``content`` block to load the
+login forms into.
+
+Some plugins, such as WebAuthn, have additional JavaScript that is necessary to
+work properly. These are dynamically loaded in via the ``extra_media`` block
+which should be located in the ``<head>`` tag::
```

Comment:
I think this is should be more generic, other (future) devices can bring additional JS and/or CSS via the `form.media` construct.

```suggestion
To create a custom template, add a template called ``two_factor/_base.html``.
As a bare minimum, its contents should contain a ``content`` block to load the
login forms into and an ``extra_media`` block located in the ``<head>`` tag.

Certain plugins can load additional static assets (CSS or JS) that are required
for the correct workings.

You can use the provided ``two_factor/_base.html`` template as a reference.
```